awx/main/conf.py

```diff
@@ -777,16 +765,6 @@ def _load_default_license_from_file():
     category=_('Logging'),
     category_slug='logging',
 )
-register(
-    'LOG_AGGREGATOR_AUDIT',
-    field_class=fields.BooleanField,
-    allow_null=True,
-    default=False,
-    label=_('Enabled external log aggregation auditing'),
-    help_text=_('When enabled, all external logs emitted by Tower will also be written to /var/log/tower/external.log.  This is an experimental setting intended to be used for debugging external log aggregation issues (and may be subject to change in the future).'),  # noqa
```

Comment:
Yep, I got rid of this in the rsyslog rewrite, because rsyslogd itself gives us the mechanisms necessary to actually log e.g., TCP and HTTP-oriented failures cc @gamuniz